Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -519,7 +519,7 @@ bool Digitizer::convertSignalsToADC(SignalContainer& signalMapCont, DigitContain
       adcs[tb] = adc;
     } // loop over timebins
     // Convert the map to digits here, and push them to the container
-    digits.emplace_back(det, row, col, adcs, getEventTime());
+    digits.emplace_back(det, row, col, adcs);
```

Comment:
The removal of the event time might be an issue for @jolopezl - please confirm.

Reply:
This came out when we originally discussed this. The above change is actually a patch from @jolopezl but he can confirm

Reply:
I confirm.